pom.xml

```diff
@@ -19,6 +19,7 @@
         <aws.sdk.version>1.12.234</aws.sdk.version>
         <log4j.version>2.17.2</log4j.version>
         <junit.version>4.13.2</junit.version>
+        <commonsIO.version>2.11.0</commonsIO.version>
```

Comment:
commons.io.verison to keep with our standard naming

Reply:
Done